base_rest/components/service.py

```diff
@@ -84,32 +66,28 @@ def _log_call(self, func, params, secure_params, res):
             _logger.debug(message, *args, extra=extra)
 
     def _get_schema_for_method(self, method_name):
-        validator_method = '_validator_%s' % method_name
+        validator_method = '_%s_request_schema' % method_name
         if not hasattr(self, validator_method):
             raise NotImplementedError(validator_method)
-        return getattr(self, validator_method)()
+        return getattr(self, validator_method)
 
     def _get_schema_output_for_method(self, method_name):
-        validator_method = '_validator_return_%s' % method_name
```

Comment:
@acsonefho `get_response_schema(self, method_name)` to be consistent with the naming of schema properties